src/components/Intro/Intro.tsx

```diff
@@ -0,0 +1,68 @@
+import React, { useRef } from 'react';
+import {
+  m, useReducedMotion, useScroll, useTransform, useWillChange, useSpring, SpringOptions,
+} from 'framer-motion';
+import { Bracket } from '../Bracket';
+import { FlexBox } from '../FlexBox';
+
+export const Intro = () => {
+  const prefersReduceMotion = useReducedMotion();
+  const willChange = useWillChange();
+  const springSetting: SpringOptions = {
+    stiffness: 130,
+    damping: 22,
+    restDelta: 0.1,
+  };
+  const sectionRef = useRef<HTMLDivElement>(null);
+  const { scrollYProgress } = useScroll({
+    target: sectionRef,
+    offset: ['-40vh', '-5vh'],
+  });
+  const scrollSpring = useSpring(scrollYProgress, springSetting);
+
+  const bracketOpacity = useTransform(scrollYProgress, [0, 1], [0, 1]);
+  // We want the text opacity to increase at a slower rate than the bracket opacity
+  const textOpacity = useTransform(scrollYProgress, [0, 0.4, 0.7, 1], [0, 0, 0.6, 1]);
+  const bracketPosition = useTransform(scrollSpring, [0, 1], ['-20vw', '0vw']);
+  const bracketRightPosition = useTransform(scrollSpring, [0, 1], ['20vw', '0vw']);
+
+  return (
+    <div ref={sectionRef} className="su-bg-saa-black su-cc su-rs-pt-10 su-rs-pb-6 su-text-white su-relative">
+      <m.div className="su-relative su-mx-auto su-w-fit" style={{ opacity: bracketOpacity }}>
+        <FlexBox justifyContent="between">
+          <m.div
+            className="su-relative su-h-auto su-shrink-0"
+            style={{ x: prefersReduceMotion ? '0' : bracketPosition, willChange }}
```

Comment:
This is how willChange is used. Also, new reduced motion strategy - keep the opacity changes but eliminate any movements (in this case, the translate x)

Reply:
This is still POC - I'll clean up the code after client decide we're actually keeping this.